### PR TITLE
Fixes "Back button of the devtool modal is not aligned properly in view"

### PR DESCRIPTION
**Pull request by yaya-usman**
Fixes:  [vector-im/element-web#21869](https://github.com/vector-im/element-web/issues/21869)

Signed-off-by: Yaya Usman [yaya-usman@users.noreply.github.com](mailto:yaya-usman@users.noreply.github.com) 

**Before:**
devtoolissue

**After:** 
devtoolissue2

Type: defect
Changes in this project generate changelog entries in element-web by default.
To suppress this:

element-web notes: none

...or to specify different notes:
element-web notes: <notes>
-->

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fixes "Back button of the devtool modal is not aligned properly in view" ([\#8381](https://github.com/matrix-org/matrix-react-sdk/pull/8381)). Contributed by @yaya-usman.

Preview: https://pr8381--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: res/css/views/dialogs/_DevtoolsDialog.scss [hunk 1]
```diff
@@ -28,7 +28,7 @@ limitations under the License.
 .mx_DevTools_content {
     margin: 10px 0;
     overflow-y: auto;
-    height: calc(100% - 124px); // 58px for buttons + 50px for header + 8px margin around
+    height: calc(100% - 130px); // 58px for buttons + 50px for header + 8px margin around
```
Comment (t3chguy) [inline res/css/views/dialogs/_DevtoolsDialog.scss:31]: Can the comment get updated?

**Review by t3chguy — CHANGES_REQUESTED** [review]

File: res/css/views/dialogs/_DevtoolsDialog.scss (hunk 1 above)
Comment (yaya-usman) [reply res/css/views/dialogs/_DevtoolsDialog.scss]: done

File: res/css/views/dialogs/_DevtoolsDialog.scss [hunk 2]
```diff
@@ -28,7 +28,7 @@ limitations under the License.
 .mx_DevTools_content {
     margin: 10px 0;
     overflow-y: auto;
-    height: calc(100% - 124px); // 58px for buttons + 50px for header + 8px margin around
+    height: calc(100% - 130px); // 58px for buttons + 50px for header + 11px margin around
```
Comment (t3chguy) [inline res/css/views/dialogs/_DevtoolsDialog.scss:31]: hmm, where is that 11px margin set? That's a really strange margin size

Comment (yaya-usman) [reply res/css/views/dialogs/_DevtoolsDialog.scss:31]: > hmm, where is that 11px margin set? That's a really strange margin size

I knew this was going to be asked😆. I seriously couldn't figure out what the previous comment actually was insinuating as well

Comment (yaya-usman) [reply res/css/views/dialogs/_DevtoolsDialog.scss:31]: Nonetheless, i realized if you summed all the values it amounts to 124px for the previous comment, so i decided to do same as well to mine to make it 130px

Comment (t3chguy) [reply res/css/views/dialogs/_DevtoolsDialog.scss:31]: No point having a comment if it is wrong/doesn't make sense.
Equally I can't reproduce the issue here

image

Comment (andybalaam) [reply res/css/views/dialogs/_DevtoolsDialog.scss:31]: I can't reproduce this in Firefox on Linux.

Comment (yaya-usman) [reply res/css/views/dialogs/_DevtoolsDialog.scss:31]: I guess i should close the issue then. It was the same across all my browsers on windows.

Comment (t3chguy) [reply res/css/views/dialogs/_DevtoolsDialog.scss:31]: my screenshot is from FF on MacOS, I can't reproduce in Element Desktop on MacOS either EXCEPT when the scrollbar kicks in but then the misalignment is in the opposing direction

image

Comment (yaya-usman) [reply res/css/views/dialogs/_DevtoolsDialog.scss:31]: I only tried it with element-web  and i am using windows. Let me close my PR and issue. Thanks

**Pull request closed without merging**